lib/widgets/content_list_view/song_tile.dart

```diff
@@ -1,9 +1,34 @@
+import 'dart:math' as math;
+
 import 'package:flutter/material.dart';
 import 'package:sweyer/sweyer.dart';
 
 /// Needed for scrollbar label computations
-const double kSongTileHeight = kSongTileArtSize + _tileVerticalPadding * 2;
 const double _tileVerticalPadding = 8.0;
+
+/// Calculate the height of one line of text rendered with the [style] and [textScaleFactor].
+double _lineHeight(TextStyle style, double textScaleFactor) {
+  final TextPainter textPainter = TextPainter(
+    text: TextSpan(text: "", style: style),
+    maxLines: 1,
+    textDirection: TextDirection.ltr,
+    textScaleFactor: textScaleFactor,
+  )..layout(minWidth: 0, maxWidth: double.infinity);
+  return textPainter.size.height;
```

Comment:
This is extremely expensive. Why do we even need a layout for that? Can't we just take the `fontSize * textScaleFactor` and that would be it?

Reply:
Unfortunately not, the values are off:
```dart
print('title height: painter=${_lineHeight(_titleTheme(theme), textScaleFactor)}, fontSize*textScale=${_titleTheme(theme).fontSize! * textScaleFactor}');
  print('ArtistWidget height: painter=${_lineHeight(ArtistWidget.defaultTextStyle(theme), textScaleFactor)}, fontSize*textScale=${ArtistWidget.defaultTextStyle(theme).fontSize! * textScaleFactor}');
```
produces
```
title height: painter=24.0, fontSize*textScale=15.0
ArtistWidget height: painter=13.0, fontSize*textScale=13.5
```
Looks like there is something else that goes into these calculations.

Reply:
I found a better solution that doesn't require a full call to layout (77cb463ac02e313812323d05072f5791576e929a).
If you are still concerned about performance, we can cache the result using the [`memoize` package](https://pub.dev/packages/memoize).